libs/ui/src/components/MediaItem/type/ImageMedia.vue

```diff
@@ -5,11 +5,15 @@
       'is-square image': !original,
       'is-detail': isDetail,
     }">
-    <img
+    <NuxtImg
       :src="src"
       class="is-block image-media__image no-border-radius"
       :alt="alt"
       data-testid="type-image"
+      loading="lazy"
```

Comment:
Missing provider ? Like in #7924  
- https://github.com/kodadot/nft-gallery/pull/7924/files#diff-01c20791d8b8c84879140e2fc9dd819eb73f19d5fc62aff1053387e6658f4607R10

Reply:
https://github.com/kodadot/nft-gallery/pull/8030/files#diff-5977891bf10802cdd3cde62f0355105a1662e65b02ae4fb404a27bb0f5f53a07R255

Reply:
I think we have Cloudflare, not IPX
- https://image.nuxt.com/providers/cloudflare
- https://image.nuxt.com/providers/ipx

Reply:
Removed nuxt-image from this PR

Reply:
> Removed nuxt-image from this PR

yeah atomic small PR preferable, feel free do it another PR if so